AdobeStockAssetApi/Api/Data/AssetInterface.php

```diff
@@ -78,6 +78,11 @@ interface AssetInterface extends \Magento\Framework\Api\ExtensibleDataInterface
      */
     const MEDIA_TYPE_ID = 'media_type_id';
 
+    /**
+     * Media gallery id
+     */
+    const MEDIA_GALLERY_ID = 'media_gallery_id';
```

Comment:
All constants should have visibility scope. Also, after refactoring to use factories instead of setter methods, probably it makes sense to remove such constants at all or move to the implementation and make them private.